src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,290 @@
+- Les éléments de Macromedia semblent qu'une grande partie disparue. Un seul élément, [`<mm:endlock>`](https://rainy-periwinkle.glitch.me/permalink/836d469b8c29e5892dedfd43556ed1b0e28a5647066858ca1c395f5b30f8485c.html), apparaît sur notre liste et sur 22 pages seulement.
+- Adobe Go-Live's [`<csscriptdict>`](https://rainy-periwinkle.glitch.me/permalink/579abc77652df3ac2db1338d17aab0a8dc737b9d945510b562085d8522b18799.html) apparaît toujours sur 640 pages de l'ensemble de données du bureau.
+- Microsoft Office's `<o:p>` L'élément apparaît toujours sur 0,5% des pages d'ordinateur, soit plus de 20 000 pages.
+
+Mais beaucoup de nouveaux arrivants ne figuraient pas non plus dans le rapport initial d'Hixie, et avec des chiffres plus importants.
+
+- [`<ym-measure>`](https://rainy-periwinkle.glitch.me/permalink/e8bf0130c4f29b28a97b3c525c09a9a423c31c0c813ae0bd1f227bd74ddec03d.html) est une balise injectée par Yandex's [Metrica analytics package](https://www.npmjs.com/package/yandex-metrica-watch). Il est utilisé sur plus de 1% des pages d'ordinateur et mobiles, consolidant ainsi sa place dans le top 100 des éléments les plus utilisés. C'est énorme!
+- [`<g:plusone>`](https://rainy-periwinkle.glitch.me/permalink/a532f18bbfd1b565b460776a64fa9a2cdd1aa4cd2ae0d37eb2facc02bfacb40c.html)à partir de la défunte Google Plus se produit sur plus de 21k pages.
+- Facebook's [`<fb:like>`](https://rainy-periwinkle.glitch.me/permalink/2e2f63858f7715ef84d28625344066480365adba8da8e6ca1a00dfdde105669a.html)  se produit sur 14k pages mobiles.
+- De même, [`<fb:like-box>`](https://rainy-periwinkle.glitch.me/permalink/5a964079ac2a3ec1b4f552503addd406d02ec4ddb4955e61f54971c27b461984.html) se produit sur 7,8k pages mobiles.
+- [`<app-root>`](https://rainy-periwinkle.glitch.me/permalink/6997d689f56fe77e5ce345cfb570adbd42d802393f4cc175a1b974833a0e3cb5.html), qui est généralement inclus dans des systèmes comme Angular, apparaît sur 8,2k pages mobiles.
+
+Comparons-les à quelques-uns des éléments HTML natifs inférieurs à la barre des 5%, en perspective.
+
+<figure id="fig9">
+  <iframe aria-labelledby="fig9-caption" width="600" height="370" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbHgqcSepZye6DrCTpifFAUYxKT1hEO56585awyMips8oiPMLYu20GETuIE8mALkm814ObJyktEe2P/pubchart?oid=962404708&amp;format=interactive"></iframe>
+    <a href="/static/images/2019/03_Markup/fig9.png" class="fig-mobile">
+      <img src="/static/images/2019/03_Markup/fig9.png" aria-labelledby="fig9-caption" width="600">
+    </a>
+  <div id="fig9-caption" class="visually-hidden">Figure 9. Popularité des éléments natifs et spécifiques au produit avec moins de 5% d'adoption.</div>
+  <figcaption>Figure 9. Popularité des éléments natifs et spécifiques au produit avec moins de 5% d'adoption.</figcaption>
+</figure>
```

Comment:
Master has new graphics format

Reply:
in the new graphics its okay to translate 'by' 'product-specific tag' ... ??

Reply:
Yes you can translate all the descriptive text, but should leave tag names and attributes untranslated. (and as an aside looks like I have a typo there in one of my "by"s!).

So I think it should look something like this (pardon any bad translation here from Google Translate - am sure you can do better!):

```
<figure id="fig9">
  <a href="/static/images/2019/03_Markup/fig9.png">
    <img src="/static/images/2019/03_Markup/fig9.png" alt="Figure 9. Popularité des éléments natifs et spécifiques au produit avec moins de 5% d'adoption." aria-labelledby="fig9-caption" aria-describedby="fig9-description" width="600" data-width="600" data-height="370" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbHgqcSepZye6DrCTpifFAUYxKT1hEO56585awyMips8oiPMLYu20GETuIE8mALkm814ObJyktEe2P/pubchart?oid=962404708&amp;format=interactive">
  </a>
  <div id="fig9-description" class="visually-hidden">
Le graphique à barres illustrant 'video' est utilisé par 184 149 sites, 'canvas' par 108 355, 'ym-measure' (une balise spécifique au produit) par 52 146, 'code' par 25 075, 'g: plusone' (une balise spécifique au produit) par 21 098, 'fb: like' (une balise spécifique au produit) par 12 773, 'fb: like-box' (une balise spécifique au produit) par 6 792, 'app-root' (une balise spécifique au produit) par 8 468, 'summary' par 6 578, template par 5 913 et un 'meter' par 0.</div>
  <figcaption id="fig9-caption">Figure 9. Popularité des éléments natifs et spécifiques au produit avec moins de 5% d'adoption.</figcaption>
</figure>
```